padrino-helpers/lib/padrino-helpers/render_helpers.rb

```diff
@@ -34,7 +34,7 @@ def partial(template, options={}, &block)
         options = { :locals => {}, :layout => false }.update(options)
         explicit_engine = options.delete(:engine)
 
-        path,_,name = template.to_s.rpartition(File::SEPARATOR)
+        path, name = File.split(template.to_s)
         template_path = File.join(path,"_#{name}").to_sym
```

Comment:
before this change if `template` was "hello", `template_path` was "hello"
after this change `template_path` is now "./hello"

This is due to how `File.split` works.

This is somewhat of a breaking change in that previously this method would call `render(explicit_engine,"hello", options)` and now it calls `render(explicit_engine,"./hello", options)`

And at least the middleman is not expecting "./hello". Should users of this gem change to expect a leading "./" or should we fix this here?